app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class UserController extends Controller
+{
+    public function get(User $user): User
+    {
+        return $user;
```

Comment:
Ik snap niet precies waar deze functie voor is

Reply:
voor het opvragen van user gegevens.